common/lib/xmodule/xmodule/x_module.py

```diff
@@ -1188,12 +1188,14 @@ def render(self, block, view_name, context=None):
         finally:
             end_time = time.time()
             course_id = getattr(self, 'course_id', '')
+            print "BLOCK_FAMILY", block.scope_ids,block_type, block.entry_point
             tags = [
                 u'view_name:{}'.format(view_name),
                 u'action:render',
                 u'action_status:{}'.format(status),
                 u'course_id:{}'.format(course_id),
-                u'block_type:{}'.format(block.scope_ids.block_type)
+                u'block_type:{}'.format(block.scope_ids.block_type),
+                u'block_family:{}'.format(block.entry_point),
```

Comment:
`block_type` is something like `vertical` or `sequential`. It's used as the XML tag, among other things. That said, in the longer view, it might be more helpful for the block_type provided at the xblock level to include both the entry_point and the id inside that entry point. It would just take some adjustment to push that through everywhere in the platform.